crates/proof-of-sql/benches/scaffold/querys.rs

```diff
@@ -60,6 +60,21 @@ const GROUPBY_COLUMNS: &[(&str, ColumnType, OptionalRandBound)] = &[
     ),
     ("c", ColumnType::Boolean, None),
 ];
+const AGGREGATE_TITLE: &str = "Aggregate";
+const AGGREGATE_SQL: &str = "SELECT SUM(a) FROM table WHERE b = a OR c = 'ab'";
```

Comment:
NIT: Can the string be anything other than "ab", just for readability, given that there are variables a and b? Really not a big deal.

Reply:
The motivation for `ab` and not something like `a` was to limit the size of the result table. Maybe there is a better way to limit the result table size?